From 85df9d1217341893ab6dbbe58fb7c878e15d832b Mon Sep 17 00:00:00 2001
From: Jakub Wilk <jwilk@jwilk.net>
Date: Wed, 24 Feb 2016 22:10:01 +0100
Subject: [PATCH] renew_cert.sh: fix unusual shebang

sh is normally lives in /bin, not in /usr/bin.
---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index dfa562c..5c83a67 100644
--- a/README.md
+++ b/README.md
@@ -171,7 +171,7 @@ for example script).
 
 Example of a `renew_cert.sh`:
 ```sh
-#!/usr/bin/sh
+#!/bin/sh
 python /path/to/acme_tiny.py --account-key /path/to/account.key --csr /path/to/domain.csr --acme-dir /var/www/challenges/ > /tmp/signed.crt || exit
 wget -O - https://letsencrypt.org/certs/lets-encrypt-x1-cross-signed.pem > intermediate.pem
 cat /tmp/signed.crt intermediate.pem > /path/to/chained.pem
